### PR TITLE
feat: Argent remote wallet support

### PR DESCRIPTION
**Pull request by JackHamer09**

### CODE REVIEW

File: src/blocks/modals/ArgentNftWarning.vue [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          this.$accessor.closeActiveModal();
+        }
+      },
+      get(): boolean {
+        return this.$accessor.currentModal !== null && this.$accessor.currentModal === "ArgentNftWarning";
+      },
+    },
+  },
+  methods: {
+    proceed() {
```
Comment (keriat) [inline src/blocks/modals/ArgentNftWarning.vue:37]: Add strict typing: :void

File: src/blocks/modals/ArgentNftWarning.vue [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+<script lang="ts">
+import Vue from "vue";
+
+export const warningCanceledKey = "canceledArgentNftWarning";
+export default Vue.extend({
+  name: "ArgentNftWarning",
+  data() {
+    return {
+      argentNftWarningCheckmark: false,
+      checkmarkDisplayed: !localStorage.getItem(warningCanceledKey),
```
Comment (keriat) [inline src/blocks/modals/ArgentNftWarning.vue:21]: Add strict typing:
```!localStorage.getItem(warningCanceledKey) as boolean```

File: src/blocks/Transaction.vue [hunk 3]
```diff
@@ -443,6 +442,12 @@ export default Vue.extend({
     if (!this.$store.getters["zk-account/loggedIn"]) {
       return;
     }
+    if ((this.mainToken === "L2-NFT" || this.type === "MintNFT") && this.$store.getters["zk-onboard/selectedWallet"] === "Argent") {
+      this.$accessor.openModal("ArgentNftWarning");
+      // @ts-ignore
+      this.$router.push(this.routeBack);
```
Comment (keriat) [inline src/blocks/Transaction.vue:448]: why `@ts-ignore`? Use `await this.$router.push` or `return this.$router.push` to avoid warnings

**Review by keriat — CHANGES_REQUESTED** [review]
Small changes requested

File: src/blocks/Transaction.vue (hunk 3 above)
Comment (JackHamer09) [reply src/blocks/Transaction.vue:447]: It's not the case how to fix it but I resolved the issue with typing

**Review by keriat — APPROVED** [review]

**Pull request merged**